privaterelay/views.py

```diff
@@ -25,14 +25,15 @@
 from oauthlib.oauth2.rfc6749.errors import CustomOAuth2Error
 from rest_framework.decorators import api_view, schema
 
-# from silk.profiling.profiler import silk_profile
-from emails.exceptions import CannotMakeSubdomainException
 from emails.models import DomainAddress, RelayAddress
 from emails.utils import incr_if_enabled
-from emails.validators import valid_available_subdomain
 
 from .apps import PrivateRelayConfig
+
+# from silk.profiling.profiler import silk_profile
```

Comment:
(suggestion, non-blocking) either delete this line or use local environment check and debug mode to enable/disable this.